robot/EDA/resources/EDA.py

```diff
@@ -394,3 +394,17 @@ def verify_selected_dropdown_value(self,**kwargs):
             actual_value = self.selenium.get_webelement(locator).text
             if not str(value).lower() == str(actual_value).lower() :
                 raise Exception (f"Drop down value in '{field}' is not updated and the value is '{actual_value}'")
+
+    def verify_dropdown_field_status(self, **kwargs):
+        """ Verify the drop down field is disabled/enabled for the user
+            we have to pass the name of the field and the status of the field 
+            True = disabled and False = enabled
```

Comment:
Per normal convention, should it be: true = enabled and false = disabled?

Reply:
Yeah! But in this case we are retrieving the value of attribute "default" so the other way. By default when the drop down field is disabled , this attribute "disabled" returns the value as True and vice versa

Reply:
Yeah but think of it from code perspective and not the functionality you are automating. What if you have to reuse this keyword for another page where default is enabled ?
I'm just trying to explain my thoughts here. You can leave it as-is, if this makes more sense to you.

Reply:
why dont u just ask to pass the status as either enabled or disabled and u convert to either true or false? that way, there is no confusion

Reply:
@Safiyab @skristem : I haven't pushed the code yet. Please take a look at the below and let me know if this is ok to be a generic keyword

![image](https://user-images.githubusercontent.com/66372160/86861803-eef64b00-c095-11ea-836a-01a5d0934703.png)